main.py

```diff
@@ -183,9 +183,12 @@ async def inner_f():
                 await bot_member.edit(nick=original_bot_nickname)
 
             # Say our goodbyes
-            await current_channel.send(
-                "Thas it y'all. Hope ya had a good **BUST** ❤️‍🔥 "
+            embed_title = "❤️‍🔥 Thas it y'all ❤️‍🔥"
+            embed_content = "Hope ya had a good **BUST**!"
```

Comment:
Should we make the `!` bold as well?

Reply:
The difference is slight but I think you're right, the bold `!` looks better
![image](https://user-images.githubusercontent.com/6956898/149640982-764d7ef0-73b1-46f1-ba53-ad0531977ea5.png)